lib/admin/actions/organizations.js

```diff
@@ -75,7 +75,7 @@ export function deleteOrganization (organization) {
   return function (dispatch, getState) {
     dispatch(deletingOrganization())
     const url = `/api/manager/secure/organization/${organization.id}`
-    return secureFetch(url, getState(), 'delete')
+    return dispatch(secureFetch(url, 'delete'))
       .then(response => response.json())
```

Comment:
Not all responses from the server are json (and so not all of them parse the json), but maybe that should just change on the server side.